tests/integration/test_cli_e2e_first_run.py

```
@@ -0,0 +1,438 @@
+    )
+
+
+def _fake_models(name: str = _SMALL_MODEL) -> list[InstalledModel]:
+    return [InstalledModel(name=name)]
+
+
+def _make_caps_mock(
+    *,
+    ollama_running: bool = True,
+    ollama_installed: bool = True,
+    models: list[Any] | None = None,
```

Ruff (0.15.13)

[error] Undefined name `Any`

(F821)